templates/packer/ansible/roles/common/tasks/packages.yml

```diff
@@ -96,7 +96,7 @@
     - rpmcopy
   ansible.posix.synchronize:
     mode: push
-    src: "{{ playbook_dir }}/roles/common/files/rpmbuild/RPMS"
```

Comment:
Cannot `{{ ansible_architecture }}` be used?

Reply:
Oh didn't know about this, gave it a quick test, it seems we can, I will update both ansible playbooks to use it!